app/kumactl/cmd/get/testdata/get/get-meshtimeout.golden.yaml

```diff
@@ -0,0 +1,23 @@
+creationTime: "0001-01-01T00:00:00Z"
+labels:
+  k8s.kuma.io/namespace: kong-mesh-system
+  kuma.io/display-name: timeout-2
+  kuma.io/mesh: default
+  kuma.io/origin: zone
+  kuma.io/zone: my-zone
+mesh: default
+modificationTime: "0001-01-01T00:00:00Z"
+name: meshtimeout-1
+spec:
+  targetRef:
+    kind: Mesh
+    proxyTypes:
+    - Gateway
+  to:
+  - default:
+      http:
+        requestTimeout: 4s
+    targetRef:
+      kind: MeshService
+      name: api-play-000_microservice-mesh_svc_8080
+type: MeshTimeout
```

Comment:
I think the original author cared about `type` being at the top / having some sort of ordering. But if we can't do that I think you can always pipe to `yq` if you care ( https://github.com/mikefarah/yq/issues/1199#issuecomment-1471145764 )

Reply:
Yeah it's imperfect. I think that's why k8s picked `kind` instead of `type`. I think this is the best we can do with low effort. We can always improve in the future